segtypes/common/code.py

```diff
@@ -61,11 +61,8 @@ def needs_symbols(self) -> bool:
         return True
 
     @property
-    def vram_end(self) -> Optional[int]:
-        if self.vram_start is not None and self.size is not None:
-            return self.vram_start + self.size + self.bss_size
-        else:
-            return None
+    def bss_size(self) -> int:
```

Comment:
if we're just storing a number in here, I think we don't need a property and can just refer to the field itself? or am I missing something